src/components/StrategyLiveChart/StrategyLiveChart.js

```diff
@@ -74,16 +89,20 @@ const StrategyLiveChart = ({
       fullscreen={fullscreenChart}
       onExitFullscreen={exitFullscreenChart}
     >
+      {/* We have to use key prop for correct Chart rerender when strategy changed
+          https://medium.com/@albertogasparin/forcing-state-reset-on-a-react-component-by-using-the-key-prop-14b36cd7448e
+      */}
       {chartMarket && (
         <Chart
           market={chartMarket}
           theme={settingsTheme}
-          layoutI={`strategy-editor-${id}`}
+          layoutI={`strategy-editor-${executionId}`}
```

Comment:
looks good, can you tell what's diff between id and executionId ?

Reply:
ok, got it from 

_Using executionId, because different strategy results could have the same (strategy) id_